Scripts/GameData.gd

```diff
@@ -83,6 +89,26 @@ func add_xp(additional_xp):
 	xp += additional_xp
 	update_xp_label()
 	emit_signal("xp_changed")
+
+func add_silver_moon():
+	var lvl = level_manager.current_level
+	var reward = int(float(lvl) / 100.0) + 1
```

Comment:
It's too complicated. The same result we should get by writing:

```
var reward = lvl / 100 + 1
```

Dividing two integers numbers gives us integer. If lvl is less than 100 reward will be 1. Otherwise bigger.